win32/build/config.w32.h.in

```diff
@@ -86,7 +86,6 @@
 # define HAVE_SETVBUF 1
 #endif
 #define HAVE_SHUTDOWN 1
-#define HAVE_SNPRINTF 1
 #define HAVE_VSNPRINTF 1
```

Comment:
Also drop HAVE_VSNPRINTF? It's not used.

Reply:
Ok, adding this one also in this one. The HAVE_VSNPRINTF symbol has been defined by configure.ac so the vsnprintf check can be also removed. This was mostly used long time ago and in some versions of oniguruma I think.